MapboxVision/src/main/java/com/mapbox/vision/utils/VideoUtils.kt

```diff
@@ -122,11 +122,16 @@ internal object VideoUtils {
             muxer.stop()
 
         } catch (e: IllegalStateException) {
-            // Swallow the exception due to malformed source.
             Log.w(TAG, "The source video file is malformed")
             return VideoProcessor.VideoPart(0, 0)
         } finally {
-            muxer.release()
+            try {
+                muxer.release()
+            } catch (e: IllegalStateException) {
+                e.printStackTrace()
```

Comment:
Do we need it?

Reply:
Why not?

Reply:
We use log here. What the reason to use printStackTrace ?